src/main/java/bio/terra/janitor/db/TrackedResourceId.java

```diff
@@ -1,15 +1,24 @@
 package bio.terra.janitor.db;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.google.auto.value.AutoValue;
 import java.util.UUID;
 
 /** Wraps the tracked_resource_id in db tracked_resource table. */
 @AutoValue
+@JsonSerialize(as = TrackedResourceId.class)
```

Comment:
add tests for json serialization of TrackedResourceId.

Reply:
done